src/helpers/index.ts

```diff
@@ -21,6 +21,13 @@ export const replaceUmlautsFromString = (str: string): string => {
     .replace(/ß/g, 'ss')
 }
 
+export const formatPoints = (num: number): string => {
```

Comment:
If you have the time you could add one or two unit tests for this one 😄

Reply:
Done ^^